package.json

```diff
@@ -42,6 +43,6 @@
     "test": "grunt run-tests",
     "lint": "grunt lint",
     "dev": "grunt dev",
-    "postinstall": "grunt githooks"
+    "postinstall": "mv ./node_modules/p5/lib/p5.js ./lib  &&  rm -rf ./node_modules/p5 &&   grunt githooks"
```

Comment:
I was just ensuring less size of repo , however it would be more meaningful if a developer would know oh yes, this p5.js has come from npm!!!.
Okay i think copying would be more convenient 👍

Reply:
Do we really need. Min version, cz i dont see  it anywhere used, like in gh-pages and tests folder it is p5. Js only not its min version

Reply:
we are using it in examples https://github.com/processing/p5.js-sound/search?q=%22p5.min.js%22&unscoped_q=%22p5.min.js%22